packages/lib/responseQueue.ts

```diff
@@ -80,6 +81,9 @@ export class ResponseQueue {
           },
           this.config.apiHost
         );
+        if (responseUpdate.displayId) {
+          await updateDisplay(responseUpdate.displayId, { responseId: response.id }, this.config.apiHost);
+        }
         if (this.surveyState.displayId) {
           markDisplayResponded(this.surveyState.displayId, this.config.apiHost);
```

Comment:
When the display has the responseId we already know that it is responded. So we can remove this code here.